feat/my/src/main/java/com/emotionstorage/my/ui/NotificationSettingScreen.kt

```
@@ -0,0 +1,169 @@
+import androidx.compose.ui.unit.dp
+import com.emotionstorage.my.ui.component.DayOfWeekSelector
+import com.emotionstorage.my.ui.component.ReminderTimeComponent
+import com.emotionstorage.my.ui.component.ToggleRow
+import com.emotionstorage.ui.component.TopAppBar
+import com.emotionstorage.ui.theme.MooiTheme
+import java.time.DayOfWeek
+import java.time.LocalTime
+
+@Composable
+fun NotificationSettingScreen(navToBack: () -> Unit) {
+}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)